### PR TITLE
docs(audit-logs): document missing attribute

### LINKED ISSUE #1159: `AuditLogEntry` documentation is missing attributes
### Summary

The documentation for the `AuditLogEntry` object does not accurately describe all attributes on the class.

### Reproduction Steps

Compare the documentation found [here](https://docs.nextcord.dev/en/stable/api.html#nextcord.AuditLogEntry) with the class itself found [here](https://github.com/nextcord/nextcord/blob/master/nextcord/audit_logs.py#L390)

### Minimal Reproducible Code

_No response_

### Expected Results

All attributes within the class are documented

### Actual Results

Attributes such as `guild` are missing from the documentation

### Intents

N/A

### System Information

Latest

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by xXenvy**
## Summary
Closes #1159

## This is a **Code Change**

- [x] I have tested my changes.
- [x] I have updated the documentation to reflect the changes.
- [x] I have run `task pyright` and fixed the relevant issues.

### CODE REVIEW

**Review by EmreTech — APPROVED** [review]

File: nextcord/audit_logs.py [hunk 1]
```diff
@@ -549,6 +548,9 @@ def created_at(self) -> datetime.datetime:
 
     @utils.cached_property
     def target(self) -> AuditTarget:
+        """:class:`Any`: The target that got changed.
```
Comment (Skelmis) [inline nextcord/audit_logs.py:551]: Can you modify this from `Any` to the actual return type please?

**Review by Skelmis — CHANGES_REQUESTED** [review]

File: nextcord/audit_logs.py (hunk 1 above)
Comment (xXenvy) [reply nextcord/audit_logs.py:551]: Yes, but this union is quite long, I don't think it is necessary.
AuditTarget = `Union[Guild, abc.GuildChannel, Member, User, Role, Invite, Emoji, StageInstance, GuildSticker, Thread, Object, AutoModerationRule, None]`

Comment (Skelmis) [reply nextcord/audit_logs.py:551]: Can you not just :class:`AuditTarget`

Comment (Skelmis) [reply nextcord/audit_logs.py:551]: I see, hrmmmm

Comment (xXenvy) [reply nextcord/audit_logs.py:551]: Yes, so I would leave it as it is

**Review by Skelmis — APPROVED** [review]

**Pull request merged**